pixart.js

```diff
@@ -1 +1,58 @@
+var pBrush = $(".brush");
+var form = $("form");
+var newColor = $("#color-field");
+var availableColors = ['#1B4370', '#1B4370', '#1B4370', '#1B4370'];
 
+// create and append swatches
+var swatch1 = $("<div></div>");
+swatch1.addClass("swatch");
```

Comment:
All of the place you have a line creating an element, and another line adding a class could be potentially combined to a single line:
```js
var swatch1 = $("<div class='swatch'></div>")
```